mailer.js

```diff
@@ -70,6 +89,7 @@ module.exports = function (log) {
     this.iosUrl = config.iosUrl
     this.supportUrl = config.supportUrl
     this.signInUrl = config.signInUrl
+    this.privacyUrl = config.privacyUrl
```

Comment:
While you are digging in here, it seems a good time to sort these in alphabetical order so they are easy to find while scanning.